src/main/java/seedu/address/model/person/TimeSlot.java

```diff
@@ -56,13 +58,26 @@ public TimeSlot(DayOfWeek day, LocalTime start, LocalTime end, Color col) {
         startTime = start;
         endTime = end;
         color = col;
+        label = DEFAULT_LABEL;
+    }
+
+    public TimeSlot(DayOfWeek day, LocalTime start, LocalTime end, String label) {
```

Comment:
Just as a suggestion, you can actually chain constructors, i.e.

`    this(day, start, end); // Calls the constructor with three variables and runs everything in that`
`    this.label = label;`